packages/functional-tests/tests/syncV3/syncV3EmailFirst.spec.ts

```diff
@@ -9,37 +9,6 @@ test.describe('Firefox Desktop Sync v3 email first', () => {
     test.slow();
   });
 
-  test('open directly to /signup page, refresh on the /signup page', async ({
```

Comment:
Q: Why the ✂️ here? No longer a valid/useful test?

Reply:
A: Yes, but Valerie explains it best.